packages/perseus/src/widgets/input-number/input-number.stories.tsx

```diff
@@ -108,3 +109,23 @@ export const Percent = (args: InputNumberOptions): React.ReactElement => {
     );
 };
 Percent.args = question3.widgets["input-number 1"].options;
+
+export const Answerful = (): React.ReactElement => {
+    const item = getAnswerfulItem("input-number", {
+        simplify: "optional",
+        size: "normal",
+        value: 42,
+    });
+    item.question.content = `The answer is 42\n${item.question.content}`;
```

Comment:
I think having the answer included in the content is a great idea. Could possibly even say what the answer field is called to help with QE testing, since that takes the longest to figure out without any context.